components/Button.tsx

```diff
@@ -0,0 +1,44 @@
+// components/Hello.tsx
```

Comment:
Esses comentários são criados automaticamente ou vc os coloca? Se for a segunda opção, creio que não seja necessário porque o VSCode recentemente liberou uma versão que mostra o caminho do arquivo aberto na tela.

Reply:
Foi criado automaticamente, mas concordo que não são necessários.